consensus/polybft/consensus_runtime.go

```diff
@@ -479,7 +374,7 @@ func (c *consensusRuntime) OnBlockInserted(fullBlock *types.FullBlock) {
 
 	// we will do PostBlock on checkpoint manager at the end, because it only
 	// sends a checkpoint in a separate routine. It doesn't do any db operations
-	if err := c.checkpointManager.PostBlock(postBlock); err != nil {
+	if err := c.bridgeManager.CheckpointManager().PostBlock(postBlock); err != nil {
 		c.logger.Error("failed to post block in checkpoint manager", "err", err)
 	}
```

Comment:
I don't like it, although I read justification in the comment above. BridgeManager would ideally know how to handle both `checkpointManager` and `stateSyncManager` internally and even don't expose it to consensus runtime (if possible).

Reply:
Fix: https://github.com/Ethernal-Tech/blade/pull/47/commits/d439dc0c3031276a74d678e4b2b915ca9fd20f74